### PR TITLE
Shelest vladimir eh1 t1

### PR DESCRIPTION
**Pull request by ShelestV**

### CODE REVIEW

File: homework/english/Vladimir_Shelest/lecture_1/Homework_1_Grammar.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+3. Does their mother cook very well?
+4. Lili is not rich, she doesn`t drive a Mercedes.
+5. Let´s go out. It`s not raining now.
+6. I have to go now. It`s getting dark
+7. A young man usually stands at the window.
+8. Ann usually makes doll dresses herself.
+9. Is his wife having coffee at the moment?
+10. Let’s go for a walk. It`s not raining.
+11. Look! The boys are coming home.
+12. Maila likes Chinese food?
```
Comment (i3oi3ka) [inline homework/english/Vladimir_Shelest/lecture_1/Homework_1_Grammar.md:15]: this is question.
```suggestion
i think
12. Does Maila likes Chinese food?
```

**Review by i3oi3ka — APPROVED** [review]
lgfm

**Review by Vladislav-Gorelchik — APPROVED** [review]

File: homework/english/Vladimir_Shelest/lecture_1/Homework_1_Grammar.md (hunk 1 above)
Comment (olgaz-skiba) [reply homework/english/Vladimir_Shelest/lecture_1/Homework_1_Grammar.md:15]: I agree with the comment

**Review by olgaz-skiba — APPROVED** [review]
Lgfm

**Review by Alex99905 — APPROVED** [review]

**Pull request merged**